devicetypes/keen-home/keen-home-smart-vent.src/keen-home-smart-vent.groovy

```diff
@@ -1,12 +1,9 @@
-/**
- *  Keen Home Smart Vent
- *
- *  Author: Keen Home
- *  Date: 2015-06-23
- */
+// keen home smart vent
+// http://www.keenhome.io
+// SmartThings Device Handler v1.0.0
 
 metadata {
-    definition (name: "Keen Home Smart Vent", namespace: "Keen Home", author: "Gregg Altschul") {
+    definition (name: "Keen Home Smart Vent", namespace: "keenhome", author: "Keen Home") {
```

Comment:
Oh oops I didn't mean to change the namespace. I was just going off our GitHub org. Let me know if you need me to revert the change.